source/common/http/status.cc

```diff
@@ -45,7 +45,7 @@ struct PrematureResponsePayload : public EnvoyStatusPayload {
 template <typename T> void storePayload(absl::Status& status, const T& payload) {
   const T* allocated = new T(payload);
   const absl::string_view sv =
-      absl::string_view(reinterpret_cast<const char*>(allocated), sizeof(allocated));
+      absl::string_view(reinterpret_cast<const char*>(allocated), sizeof(*allocated));
```

Comment:
this looks like it fixes the issue but would sizeof(T) compile?  Seems a bit easier to read that way to me.
/wait-any